multi/imxrt-multi/rtt.c

```diff
@@ -0,0 +1,305 @@
+/*
+ * Phoenix-RTOS
+ *
+ * iMX RT RTT communication driver
+ *
+ * Copyright 2017-2024 Phoenix Systems
+ * Author: Kamil Amanowicz, Marek Bialowas, Aleksander Kaminski, Jacek Maksymowicz
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include <errno.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <sys/threads.h>
+#include <sys/file.h>
+
+#include <librtt.h>
+
+#include "common.h"
+#include "rtt.h"
+
+
+#define RTT_MAP_NAME             "rtt"
```

Comment:
This could be `#ifndef`ed to make it possible to change the map name via e.g. board_config.h.

Reply:
Done, renamed to `RTT_SYSPAGE_MAP_NAME` to keep in sync with your changes in kernel.